contestant_containers/arch/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM docker.io/blackarchlinux/blackarch:latest
+LABEL maintainer "Trevor Bryant <@apporima>"
+
+# Temporary patch to build on Ubuntu 20.04/10 & GitHub CI
+# https://bugs.archlinux.org/index.php?do=details&task_id=69563
+RUN patched_glibc=glibc-linux4-2.33-4-x86_64.pkg.tar.zst && \
+    curl -LO "https://repo.archlinuxcn.org/x86_64/$patched_glibc" && \
+    bsdtar -C / -xvf "$patched_glibc"
+
+# Install packages
+RUN pacman -Syu --noconfirm git make gcc pkgconf which python-pip wget openssh vim tmux screen supervisor iw mlocate man pciutils less bash-completion xorg-server-xvfb x11vnc dialog xfce4 xfce4-goodies xfce4-power-manager inetutils \
+# rfctf tools
+    hostapd wpa_supplicant bully wireshark-qt wireshark-cli aircrack-ng mdk3 mdk4 freeradius asleap hostapd-wpe wifite pixiewps reaver hashcat hashcat-utils hcxtools hcxdumptool
+
+# Install packages from source
+# noVNC -- copied from https://github.com/ponsfrilus/arch-novnc
+WORKDIR /opt/
+RUN git clone https://github.com/kanaka/noVNC.git
+# Avoid another checkout when launching noVnc
+WORKDIR /opt/noVNC/utils/
+RUN git clone https://github.com/kanaka/websockify
```

Comment:
Arch seems to have a novnc package?
https://aur.archlinux.org/packages/novnc/

Reply:
I've had trouble installing this from AUR as that appears an external repository with a different installation method; `makepkg`. There are a few recommended options that I need to try.

Reply:
thank you so much, that really helps.  (plus it's super cool for demos and stuff)

Reply:
As promised, [I have added](https://github.com/BlackArch/blackarch/commit/939d37af65a7cff66d2ce3b9a468f25dea141f3a) `novnc` to our repos and tested it to be compatible with the existing `websockify` package. From now you just have to add `novnc` to the list of packages, remove the cloning steps and everything novnc-related should work out of the box.

Reply:
Eduard, thanks so much. That was a very fast turn out and I appreciate it.